.github/dependabot.yml

```diff
@@ -82,19 +82,19 @@ updates:
         versions:
           - '>= 4'
 
-  #   - package-ecosystem: bundler
-  #     directory: "/packages/ruby"
-  #     schedule:
-  #       interval: monthly
-  #     allow:
-  #       - dependency-type: production
-  #     open-pull-requests-limit: 10
-  #     labels:
-  #       - dependencies
-  #       - ruby
-  #     commit-message:
-  #       prefix: chore(deps)
-  #       prefix-development: chore(deps-dev)
+  - package-ecosystem: bundler
```

Comment:
I don't believe this is related to the other changes. If not, given we squash and merge commits from PRs, it probably makes sense to move this to a separate PR so we have a more distinct commit history post-merge.